ensembl/conf/ini-files/salmo_salar.ini

```
@@ -58,13 +58,13 @@ SPECIES_RELEASE_VERSION = 31
 
 [SAMPLE_DATA]
 
-VARIATION_PARAM = 1:1343602:G_C:PRJEB34225;vf=1:1343602:G_C:PRJEB34225
-VARIATION_TEXT  = 1:1343602:G_C:PRJEB34225
+VARIATION_PARAM  = 1:916750:C_G:PRJEB34225;vf=1:916750:C_G:PRJEB34225
+VARIATION_TEXT   = 1:916750:C_G:PRJEB34225
 
-VEP_HGVS        = ENSSSAT00000000041.1:c.4A>G\nENSSSAT00000009226.1:c.248+16T>C\nENSSSAT00000092070.1:c.5del 
-VEP_VCF         = 1 307025  . A G . . .\n9 73967 . T C . . .\n10 497538  . GG  G . . .
-VEP_SPDI        = 1:307024:A:G\n9:73966:T:C\n10:497538:G:
-VEP_ENSEMBL     = 1 307025  307025  A/G 1\n9 73967 73967 T/C 1\n10 497539  497539  G/- 1
+VEP_HGVS         = ENSSSAT00000000041.2:c.1091G>A\nENSSSAT00000039902.2:c.219C>T\nENSSSAT00000120111.2:c.*731+8G>C
+VEP_VCF          = 1 2449713 . C T\n10 2232401 . C T\n27 9534658 . C G
+VEP_SPDI         = 1:2449713:C:T\n10:2232401:C:T\n27:9534658:C:G
```

> In SPDI notation, the position refers to the base before the variant, not the base of the variant itsef.

So, correct example would be -

```suggestion
VEP_SPDI         = 1:2449712:C:T\n10:2232400:C:T\n27:9534657:C:G
```